src/test/java/org/cryptomator/cryptofs/health/dirid/DirIdCheckTest.java

```diff
@@ -130,7 +143,11 @@ private static void initDirStructure(Path root, String line) throws UncheckedIOE
 				var file = line.substring(0, sep);
 				var contents = line.substring(sep + 3);
 				Files.createDirectories(root.resolve(file).getParent());
-				Files.writeString(root.resolve(file), contents, StandardCharsets.US_ASCII, StandardOpenOption.CREATE_NEW);
+				if (!contents.equals("EMPTY")) {
```

Comment:
1. please invert this if
2. not sure if `EMPTY` is a sufficiently indicating that this is a magic value. Maybe use `[EMPTY]` or something that differentiates more from "normal" contents

Reply:
Isn't `var contents` already an empty string, if nothing follows after `sep`?

Reply:
Nope, because seperator is " = " (note the leading and trailing whitespaces). A text block line is ended with a non-whitespace character, thus writing nothing would lead to a line ending with " =" and the splitting would fail.

There are workarounds for this, e.g. https://awesomely-java.blogspot.com/2021/03/techniques-for-including-trailing.html.

I would go with [EMPTY]. It's our code, we don't need the best approach, only a suffiecently future-proof and easy-to-understand one.